src/pages/dataTableUtilities/getColumns.js

```diff
@@ -41,20 +41,22 @@ const PAGE_COLUMN_WIDTHS = {
   [ROUTES.SUPPLIER_REQUISITIONS]: [1.5, 2, 1, 1, 1, 1],
   [ROUTES.PRESCRIPTION]: [2, 4, 2, 2, 1],
   [ROUTES.PRESCRIPTIONS]: [1.5, 2.5, 2, 1.5, 3, 1],
-  fridges: [1, 3],
-  sensors: [1, 3, 2, 2],
+  fridges: [1, 3, 1],
+  sensors: [3, 3, 1, 1, 1],
   [TABS.ITEM]: [1, 3, 1],
   [TABS.PRESCRIBER]: [3, 3, 1],
 };
 
 const PAGE_COLUMNS = {
   sensors: [
-    COLUMN_NAMES.LOCATION,
     COLUMN_NAMES.EDITABLE_NAME,
+    COLUMN_NAMES.LOCATION,
     COLUMN_NAMES.BATTERY_LEVEL,
     COLUMN_NAMES.MAC_ADDRESS,
+    COLUMN_NAMES.REMOVE,
   ],
-  fridges: [COLUMN_NAMES.CODE, COLUMN_NAMES.DESCRIPTION],
+  fridges: [COLUMN_NAMES.EDITABLE_CODE, COLUMN_NAMES.DESCRIPTION, COLUMN_NAMES.REMOVE],
```

Comment:
adding pretty basic remove logic to sensors and fridges for now. I think some more detailed logic might be needed, given this could leave orphan records. Since it's store data, I think dissallowing deletion when there are no related records would be all good